ansible_rulebook/app.py

```diff
@@ -92,6 +92,8 @@ async def run(parsed_args: argparse.Namespace) -> None:
         startup_args.controller_url = parsed_args.controller_url
         startup_args.controller_token = parsed_args.controller_token
         startup_args.controller_ssl_verify = parsed_args.controller_ssl_verify
+        startup_args.source_dir = parsed_args.source_dir
```

Comment:
@benthomasson Can these go into the [settings](https://github.com/ansible/ansible-rulebook/blob/eff4dbb0fc1110e0aa8907906ec2b46d5e6453d7/ansible_rulebook/conf.py#L27) and it can be accessed from anywhere. We have been moving most of the passed in args into the settings.